students/chan/westagram/users/models.py

```diff
@@ -0,0 +1,18 @@
+from django.db        import models
+from django.db.models import Model
+
+# Create your models here.
+class User(models.Model):
+    name                     = models.CharField(max_length=45)
+    email                    = models.EmailField(max_length=128, null=False, unique=True)
```

Comment:
null=False는 기본값이므로 전부 제거해 주세요.
```suggestion
    email                    = models.EmailField(max_length=128, unique=True)
```

Reply:
1. null = False 제거 완료하였습니다.
2. persnal_site 의 max_length 1000으로 수정 하였습니다.
3. null = True, Blank = True와의 차이점 공부 및 코멘트 작성하였습니다.